packages/edit-navigation/src/components/menu-editor/use-navigation-blocks.js

```diff
@@ -168,9 +79,7 @@ const menuItemsToLinkBlocks = (
 	};
 
 	// menuItemsToTreeOfLinkBlocks takes an array of top-level menu items and recursively creates all their innerBlocks
-	const linkBlocks = menuItemsToTreeOfLinkBlocks(
-		itemsByParentID[ 0 ] || []
-	);
+	const linkBlocks = menuItemsToTreeOfBlocks( itemsByParentID[ 0 ] || [] );
```

Comment:
We might want to rename `linkBlocks` on the basis that in the future we could have other kinds of blocks coming in.